rero_mef/agents/cli.py

```diff
@@ -0,0 +1,256 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of RERO MEF.
+# Copyright (C) 2018 RERO.
+#
+# RERO MEF is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# RERO MEF is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with RERO MEF; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, RERO does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Click command-line interface for MEF record management."""
+
+from __future__ import absolute_import, print_function
+
+import itertools
+from time import sleep
+
+import click
+from celery.bin.control import inspect
+from flask.cli import with_appcontext
+
+from .tasks import \
+    create_mef_and_agents_from_viaf as task_mef_and_agents_from_viaf
+from .tasks import create_mef_from_agent as task_mef_from_agent
+from ..mef.api import MefRecord
+from ..utils import get_agent_class, get_agent_classes, progressbar
+from ..viaf.api import ViafRecord
+
+
+@click.group()
+def utils():
+    """Misc management commands."""
+
+
+def queue_count():
+    """Count tasks in celery."""
+    inspector = inspect()
+    task_count = 0
+    reserved = inspector.reserved()
+    if reserved:
+        for key, values in reserved.items():
+            task_count += len(values)
+    active = inspector.active()
+    if active:
+        for key, values in active.items():
+            task_count += len(values)
```

Comment:
could be replaced with task_count = sum(active.values())

Reply:
Thank you learned a new python function.